modules/ROOT/pages/index.adoc

```diff
@@ -205,63 +215,74 @@ Please reconnect.
-| The statement refers to a parameter that was not provided in the request. 
+| The statement refers to a parameter that was not provided in the request.
 
 | Neo.ClientError.Statement.PropertyNotFound
-| The statement refers to a non-existent property. 
+| The statement refers to a non-existent property.
 
 | Neo.ClientError.Statement.RuntimeUnsupportedError
-| This query is not supported by the chosen runtime. 
+| This query is not supported by the chosen runtime.
 
 | Neo.ClientError.Statement.SemanticError
-| The statement is syntactically valid, but expresses something that the database cannot do. 
+| The statement is syntactically valid, but expresses something that the database cannot do.
 
 | Neo.ClientError.Statement.SyntaxError
-| The statement contains invalid or unsupported syntax. 
+| The statement contains invalid or unsupported syntax.
 
 | Neo.ClientError.Statement.TypeError
-| The statement is attempting to perform operations on values with types that are not supported by the operation. 
+| The statement is attempting to perform operations on values with types that are not supported by the operation.
 
 | Neo.ClientError.Statement.UnsupportedAdministrationCommand
-| This administration command is not supported. 
+| This administration command is not supported.
+
+| Neo.ClientError.Statement.UnsupportedOperationError
+| This query preformed an operation that is not supported in this context.
 
 | Neo.ClientError.Transaction.ForbiddenDueToTransactionType
 |
 The transaction is of the wrong type to service the request.
-For instance, a transaction that has had schema modifications performed in it cannot be used to subsequently perform data operations, and vice versa. 
+For instance, a transaction that has had schema modifications performed in it cannot be used to subsequently perform data operations, and vice versa.
 
 | Neo.ClientError.Transaction.InvalidBookmark
 |
 Supplied bookmark cannot be interpreted.
 You should only supply a bookmark that was previously generated by Neo4j.
-Maybe you have generated your own bookmark, or modified a bookmark since it was generated by Neo4j. 
+Maybe you have generated your own bookmark, or modified a bookmark since it was generated by Neo4j.
 
 | Neo.ClientError.Transaction.InvalidBookmarkMixture
 |
 Mixing bookmarks generated by different databases is forbidden.
 You should only chain bookmarks that are generated from the same database.
-You may however chain bookmarks generated from system database with bookmarks from another database. 
+You may however chain bookmarks generated from system database with bookmarks from another database.
+
+| Neo.ClientError.Transaction.LockClientStopped
+| The transaction has been terminated, so no more locks can be acquired.
+This can occur because the transaction ran longer than the configured transaction timeout,
+or because a human operator manually terminated the transaction, or because the database is shutting down.
```

Comment:
```suggestion
This can occur because the transaction ran longer than the configured transaction timeout, or because a human operator manually terminated the transaction, or because the database is shutting down.
```